Workade/Views&ViewModels/CheckList/CheckListDetail/CheckListBottomSheet/CheckListBottomSheetViewController.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  CheckListBottomSheetViewController.swift
+//  Workade
+//
+//  Created by Wonhyuk Choi on 2022/10/25.
+//
+
+import UIKit
+
+class CheckListBottomSheetViewController: UIViewController {
+    var defaultHeight: CGFloat = 280
+    
+    private let dimmedView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .clear
+        
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
```suggestion
        view.backgroundColor = .clear
        view.translatesAutoresizingMaskIntoConstraints = false
```
[P3] 필요 없는 공백 줄여주세요.